Android/app/src/test/java/utils/DateUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+package utils;
+
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.Locale;
+import java.util.logging.SimpleFormatter;
+
+import static org.junit.Assert.assertEquals;
+
+public class DateUtilsTest {
+
+    @Test
+    public void rfc3339ToMills() {
+        assertEquals(1572374198000L , DateUtils.rfc3339ToMills("2019-10-29T18:36:38.000GMT").longValue());
+    }
+    @Test
+    public void getDate_with_mils() {
+        assertEquals("29 Oct 2019" , DateUtils.getDate(1572374198000L));
```

Comment:
Can you also add some negative test cases for invalid dates

Reply:
This isnt resolved yet?

Reply:
How is it resolved now?

Reply:
@fokion Why are you marking this as resolved?

Reply:
https://github.com/project-travel-mate/Travel-Mate/commit/1b86d19bcb956af2745d37bfb6f4d117b3c2512c

Reply:
that is the reason why I resolve it. there are two sets of invalid tests that cover a wrong date format and a text that is invalid as it is not a date at all.